x11-libs/libnotify/files/debian-autopkgtest.patch

```diff
@@ -0,0 +1,86 @@
+This patch adapts the debian autopkgtest program and specifies it as a meson test
```

Comment:
Just want to say that I really like this idea of trying to have _something_ useful in src_test, and we should consider doing it more often (even if we just write a tiny test program in src_test in some cases and test if it links, etc).